app/react/components/BoxSelector/common-options/build-methods.tsx

```diff
@@ -49,6 +50,16 @@ export const customTemplate: BoxSelectorOption<'template'> = {
   value: 'template',
 };
 
+export const helm: BoxSelectorOption<'helm'> = {
+  id: 'method_helm',
+  icon: Helm,
+  iconType: 'raw',
+  label: 'Helm',
+  description: 'Use a Helm chart',
```

Comment:
Shouldn't this be just "Helm chart" ?   I'm going by the mockup.  Assuming this is what you see in the box selector